### DISCUSSION TITLE
Possibility to includes hover event into Y axis

### DISCUSSION THREAD

**Discussion by gdoCaetano** [plouc/nivo#1911] [Q&A] [answered]
Hello!

I would like to know if it's possible to trigger a hover event on the Y axis when hovering over it.

That is, I would like to trigger a tooltip on the Y axis to display the results displayed on the X axis.

In the example below, when positioning the mouse on "AM", display the same behavior that is triggered when I position the mouse over the value "200". In this case, the result with the caption would bring all the resulting values ​​on the X axis into this event.

![image](https://user-images.githubusercontent.com/30940467/151431451-ee954d2e-8e38-443c-b3c4-9d18516a71a4.png)

Thanks a lot! :smile:

**Comment by plouc** [comment, marked as answer]
Hello,

No, it's not possible, some people also requested a stacked tooltip for the x axis (showing all values for X), but that's not something I'm planning to add soon.

If you want to contribute, there's an old PR for showing all the values for the same axis (https://github.com/plouc/nivo/pull/70), however this package changed a lot since, not sure how easy it would be to re-use.

**Reply by gdoCaetano** [reply]
Thanks!!

I'll take a look :D
